.github/dependabot.yml

```diff
@@ -8,7 +8,23 @@ updates:
     timezone: Europe/Berlin
   open-pull-requests-limit: 20
   ignore:
-  # Substrate (+ Polkadot/Cumulus pallets) dependencies
+  # Bridges polkadot-sdk dependencies
+  - dependency-name: bp-*
+    versions:
+      - ">= 0"
+  - dependency-name: bridge-runtime-common
+    versions:
+      - ">= 0"
+  - dependency-name: parachains-relay
```

Comment:
We have no other `relay-*` dependencies here? `finality-relay`, `messages-relay`, ...?

Reply:
You're right. Good catch. Added the other relays as well.